Dockerfile

```diff
@@ -27,7 +27,7 @@ ARG PYMOD_VERSION=0.0.0
 
 WORKDIR /src/
 COPY cray_product_catalog/ ./cray_product_catalog
-COPY setup.py requirements.txt constraints.txt README.md ./
+COPY setup.py requirements.txt constraints.txt requirements-pyyaml.txt requirements-non-pyyaml.txt README.md ./
```

Comment:
Can you add a detailed comment here so that we know why you did this and what it will take to remove the work-around?

I know the tracks are in the PR, but I would rather not have to spend 30 min digging through git history 2 years from now when we need to know.

Reply:
Done. Let me know what you think.